Adyen.xcodeproj/project.pbxproj

```diff
@@ -5396,6 +5404,7 @@
 		};
 		F9A4A303237323E700D774E8 /* Format and lint */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
what's this for? :-)

Reply:
this was causing some odd warning in Xcode 14.